repo_manager/gh/branch_protections.py

```diff
@@ -309,7 +311,12 @@ def check_repo_branch_protections(
             diff_protections[config_bp.name] = ["Branch is not protected"]
             continue
 
-        this_protection = repo_bp.get_protection()
+        try:
```

Comment:
@andrewthetechie this is where #51 is occurring, and has been since I first opened it.